client_generic/Networking/Networking.cpp

```diff
@@ -26,9 +26,11 @@ CCurlTransfer::CCurlTransfer( const std::string &_name ) : m_Name( _name ), m_St
 	m_pCurlM = curl_multi_init();
 	if( !m_pCurlM )
 		g_Log->Info( "Failed to init curl multi instance." );
-		
+
+#if defined(_MSC_VER) && (_MSC_VER != 1900)		

```

Comment:
Why this ? It is a common multi-platform code, it should not be compiled only on Windows. Please change it to something more open.